packages/superset-ui-preset-chart-xy/src/utils/XYChartLayout.tsx

```diff
@@ -154,7 +154,7 @@ export default class XYChartLayout {
 
     return axis && this.xLayout ? (
       <XAxis
-        label={axis.getTitle()}
+        label={axis.getTitle() || ''}
```

Comment:
The logic here is bit complex. Can we make `axis.getTitle()` return `''` when there is no title, and for `hasTitle`, we can just test if `axis.getTitle() === ''`. I think it will make the code cleaner.

Reply:
yep done